Bio/UniProt/GOA.py

```diff
@@ -118,9 +120,7 @@
     'Taxon',
     'Parent_Object_ID',
     'DB_Xref',
-    'Gene_Product_Properties',
-    'Annotation_Target_Set',
-    'GO_Annotation_Complete']
+    'Gene_Product_Properties']
```

Comment:
Was this an oversight in the original, and the removed fields were GPI 1.0 only?

Reply:
I think so, the documentation  for GPI 1.0 and GPI 1.1 is no longer available only for GPI 1.2
http://geneontology.org/docs/gene-product-information-gpi-format/
the file, gp_information.goa_yeast.28.gpi, added here dated 2013-11-11, includes the fields on its header, that is what I have used as a reference.